README.md

```diff
@@ -111,12 +127,15 @@ pip install -r requirements/base.txt
 
 (TODO: investigate using ``python3 -m venv `pwd`/venv`` instead)
 
-Quick & dirty test:
+Copy `src/cloudlabs/secrets.example.py` to `src/cloudlabs/secrets.py` and fill in values.
 
+Install Terraform:
 ```bash
-cd src
-sudo apache2ctl stop
-sudo FLASK_APP=autoapp.py /home/cloudlabs/user-portal/venv/bin/flask run --host=0.0.0.0 --port=80
+cd
+curl -sSL -o terraform.zip "https://releases.hashicorp.com/terraform/0.10.2/terraform_0.10.2_linux_amd64.zip"
```

Comment:
Latest version is now https://releases.hashicorp.com/terraform/0.10.6/terraform_0.10.6_darwin_amd64.zip?_ga=2.201597522.234164828.1505888909-222463015.1499336522. I hope there's a better way to install it sometime soon so it's easy to keep updated. I only know because I once asked a question in their gitter and keep on getting notifications from there.

Reply:
Yeah, it's a pain. Similarly with the Azure provider which gets downloaded when you run `terraform init` in a folder with a .tf file using Azure...